simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/FOSS375CatalogZero.scala

```diff
@@ -0,0 +1,277 @@
+package com.puppetlabs.gatling.node_simulations
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.jdbc.Predef._
+
+import com.puppetlabs.gatling.runner.SimulationWithScenario
+import org.joda.time.LocalDateTime
+import org.joda.time.format.ISODateTimeFormat
+
+class FOSS375CatalogZero extends SimulationWithScenario {
+
+	val httpProtocol = http
```

Comment:
I don't *think* these matter, as I think that our subclass and driver override them anyway.  (I think it would be pretty easy to tell if they didn't, because the behavior of the sim would be very different... but it's been a long time since I looked into that code.)